Tests/CloudLibClientTests/NamespaceComparers.cs

```diff
@@ -0,0 +1,75 @@
+using System.Collections.Generic;
+using System.Diagnostics.CodeAnalysis;
+using Opc.Ua.Cloud.Library.Client;
+
+namespace CloudLibClient.Tests
+{
+    internal class OrganisationComparer : IEqualityComparer<Organisation>
+    {
+        public bool Equals(Organisation x, Organisation y)
+        {
+            if (x == null && y == null) return true;
+            if (x == null) return false;
+            if (y == null) return false;
+            return x.Name == y.Name
+                && (x.Description == y.Description || (string.IsNullOrEmpty(x.Description) && string.IsNullOrEmpty(y.Description)))
+                && x.LogoUrl?.ToString() == y.LogoUrl?.ToString()
+                && (x.ContactEmail == y.ContactEmail || (string.IsNullOrEmpty(x.ContactEmail) && string.IsNullOrEmpty(y.ContactEmail)))
+                && x.Website?.ToString() == y.Website?.ToString();
+        }
+
+        public int GetHashCode([DisallowNull] Organisation obj)
+        {
+            throw new System.NotImplementedException();
```

Comment:
Is this allowed?

Reply:
It's used just in the test, and the tests only need the Equals comparison.